www/include/configuration/configServers/popup/central.yaml

```diff
@@ -0,0 +1,59 @@
+name: gorgoned-__SERVERNAME__
+description: Configuration for remote server __SERVERNAME__
+gorgone:
+  gorgonecore:
+    privkey: "/var/lib/centreon-gorgone/.keys/rsakey.priv.pem"
+    pubkey: "/var/lib/centreon-gorgone/.keys/rsakey.pub.pem"
+  modules:
+    - name: httpserver
+      package: "gorgone::modules::core::httpserver::hooks"
+      enable: true
+      address: "0.0.0.0"
+      port: "8085"
+      ssl: false
+      auth:
+        enabled: false
+      allowed_hosts:
+        enabled: true
+        subnets:
+          - 127.0.0.1/32
+    - name: action
+      package: "gorgone::modules::core::action::hooks"
+      enable: true
+    - name: cron
+      package: "gorgone::modules::core::cron::hooks"
+      enable: true
+      cron: !include cron.d/*.yaml
+    - name: nodes
+      package: "gorgone::modules::centreon::nodes::hooks"
+      enable: true
+    - name: proxy
+      package: "gorgone::modules::core::proxy::hooks"
+      enable: true
+    - name: legacycmd
+      package: "gorgone::modules::centreon::legacycmd::hooks"
+      enable: true
+      cmd_dir: "/var/lib/centreon/centcore/"
```

Comment:
shouldn't these two path be retrieved from current configuration ? ie : source installation